cli/src/commands/build.js

```diff
@@ -13,6 +13,10 @@ const makeShell = require('../lib/shell')
 const { validatePackage } = require('../lib/validatePackage')
 const { handler: pack } = require('./pack.js')
 
+process.on('unhandledRejection', err => {
```

Comment:
We can add a comment explaining why we do this. It would be great if we could introduce this in a single place in the CLI Engine so we don't have to do this for all our CLI tools. Currently looking into it.

Reply:
A comment sounds good -- could maybe link to the [comment](https://jira.dhis2.org/browse/CLI-38?focusedCommentId=45563&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-45563) you added to the Jira issue, or even add that info to the PR and link here which I think I like better

Reply:
@varl we should be able to do it here I think?  https://github.com/dhis2/cli-helpers-engine/blob/master/lib/makeEntryPoint.js

Reply:
@amcgee that's right, I've tested that it works for this case. Wanted to test a bit more in dhis2/cli. Will post a new PR in engine and then link it to this one after verification.

Reply:
Confirmed, works in other places too. Here's the PR:

https://github.com/dhis2/cli-helpers-engine/pull/145